.github/workflows/codeql.yml

```diff
@@ -15,7 +24,16 @@ on:
 
 jobs:
   codeql:
-    runs-on: ubuntu-20.04
+    runs-on: 'ubuntu-latest'
+    timeout-minutes: 360
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+    
+    container:
+      image: 'docker-cli-dev:latest'
```

Comment:
Oh actually you need to remove this otherwise it fails:

![image](https://github.com/docker/cli/assets/1951866/a0d38b83-33b9-48bd-b8c3-a7cd38b1185d)

Don't think we need it for codeql

Reply:
Oh yeah, that wasn't supposed to go into the commit - fixed!